Server/npm-debug.log

```diff
@@ -1,23 +0,0 @@
-0 info it worked if it ends with ok
```

Comment:
add this file to `.gitignore` and then delete it, so git does not keep track of it